draft-ietf-quic-http.md

```diff
@@ -552,10 +553,11 @@ the interpretation of the associated Element ID fields.
 Note that the root of the tree cannot be referenced using a Stream ID of 0, as
 in {{!RFC7540}}; QUIC stream 0 carries a valid HTTP request.  The root of the
 tree cannot be reprioritized. A PRIORITY frame sent on a request stream that
-prioritizes any other stream MUST be treated as a stream error of type
-HTTP_MALFORMED_FRAME.  Likewise, a PRIORITY frame sent on a control stream that
-prioritizes the current stream MUST be treated as a connection error of type
-HTTP_MALFORMED_FRAME.
+prioritizes any other stream or expresses a dependency on a request with a
```

Comment:
I am not sure if we need "prioritizes any other stream or".

Now that exclusive flag is going away, is there any case that could hit this condition?

Reply:
This is a perhaps-unclear reference to a PRIORITY frame where `Prioritized Element Type` is not `11` (i.e. the thing being prioritized is anything other than "this stream").

Reply:
Ah. Thank you for the clarification. I missed that.

My preference goes to stating that `Prioritized Element Type` MUST be `11` when the frame is sent on a request stream and that it MUST NOT be `11` when the frame is sent on a control stream.

This is because I prefer to avoid having multiple representation that means the same; under the current provision, it seems that PEID could be encoded in two ways for a PRIORITY frame sent on a request stream; either set Prioritized Element Type to `11` or set it to `00` and set Prioritized Element ID to the ID of the stream. I fear that we might see interoperability issues in the wild if _most_ of the clients use the `11` form but some used the `00` form.

Reply:
Good point.  The requirement for the values was already there (in the definition of Prioritized Element Type), but I'll scope the required-error language to the specific values as well.